Xamarin.Forms.Platform.iOS/Renderers/ButtonLayoutManager.cs

```diff
@@ -211,6 +217,38 @@ internal void UpdateText()
 			Control.SetAttributedTitle(disabled, UIControlState.Disabled);
 
 			UpdateEdgeInsets();
+			UpdateLineBreakMode();
```

Comment:
I think it would make sense to have this line in the `Update` method from `ButtonLayoutManager`.

Reply:
Yeah you're right, my mistake on that 😅